tableauserverclient/server/request_factory.py

```diff
@@ -238,21 +238,46 @@ def add_user_req(self, user_id):
         user_element.attrib['id'] = user_id
         return ET.tostring(xml_request)
 
-    def create_req(self, group_item):
+    def create_local_req(self, group_item):
         xml_request = ET.Element('tsRequest')
         group_element = ET.SubElement(xml_request, 'group')
         group_element.attrib['name'] = group_item.name
+        if (group_item.license_mode is not None):
```

Comment:
Nit: no `()` around if statements (unless it's multi-line is the general rule, but I also think nested is ok)